packages/shared/lib/contexts/governance/utils/getParticipationsForProposal.ts

```diff
@@ -3,10 +3,8 @@ import type { ParticipationOverview, TrackedParticipationOverview } from '@iota/
 import { allParticipationOverviews } from '@contexts/governance/stores'
 import { selectedWalletId } from 'shared/lib/core/wallet'
 
-export function getParticipationsForProposal(
-    proposalId: string,
-    walletId = get(selectedWalletId)
-): { [outputId: string]: TrackedParticipationOverview } {
+export function getParticipationsForProposal(proposalId: string): { [outputId: string]: TrackedParticipationOverview } {
+    const walletId = get(selectedWalletId)
```

Comment:
Why not pass the `walletId` in the function?
the same for the rest of the `proposals` functions

Reply:
Because we weren't using that parameter :joy: If we ever need it back (I doubt it) we can bring it back